Okta.AspNetCore/OpenIdConnectOptionsHelper.cs

```diff
@@ -76,6 +78,15 @@ public static void ConfigureOpenIdConnectOptions(OktaMvcOptions oktaMvcOptions,
                 oidcOptions.ClaimActions.Add(new MapAllClaimsAction());
             }
 
+            if (oktaMvcOptions.AddFullyQualifiedIdentityClaimNames)
```

Comment:
Would it be better if we do this by default instead of adding another config param?

Reply:
The param is true by default. I don't expect any issues with these additional claim actions but still it's better to be able to revert to the original actions.